### PR TITLE
BF: allow for BrainModels or Parcels  to contain a single vertex

### PR DESCRIPTION
**Pull request by MichielCottaar**
Fixes an edge case in the reading of CIFTI files.
Tests for this edge case have been added for both the BrainModel and Parcel axes.

### CODE REVIEW

File: nibabel/cifti2/parse_cifti2.py [hunk 1]
```diff
@@ -517,7 +517,7 @@ def flush_chardata(self):
             # conversion to numpy array
             c = BytesIO(data.strip().encode('utf-8'))
             vertices = self.struct_state[-1]
-            vertices.extend(np.loadtxt(c, dtype=np.int))
+            vertices.extend(np.atleast_1d(np.loadtxt(c, dtype=np.int)))
```
Comment (effigies) [inline nibabel/cifti2/parse_cifti2.py]:
```suggestion
            vertices.extend(np.loadtxt(c, dtype=np.int, ndmin=1))
```

File: nibabel/cifti2/parse_cifti2.py [hunk 2]
```diff
@@ -531,7 +531,7 @@ def flush_chardata(self):
             # conversion to numpy array
             c = BytesIO(data.strip().encode('utf-8'))
             index = self.struct_state[-1]
-            index.extend(np.loadtxt(c, dtype=np.int))
+            index.extend(np.atleast_1d(np.loadtxt(c, dtype=np.int)))
```
Comment (effigies) [inline nibabel/cifti2/parse_cifti2.py]:
```suggestion
            index.extend(np.loadtxt(c, dtype=np.int, ndmin=1))
```

**Comment by MichielCottaar** [comment]
Well spotted, that indeed has the same effect (sorry, I only noticed that I could simply have accepted your suggested changes after pushing a commit based on your update)

**Comment by effigies** [comment]
LGTM. Thanks.
